READ_BEFORE_UPDATE.md

```diff
@@ -1,5 +1,38 @@
 # Update Notes
 
+## Migrating from privacyIDEA 3.9.2 to eduMFA
+
+> [!CAUTION]
+>  * Make a backup of your existing installation! This backup should include the config files, the encryption keys and
+     the database.
+>  * Ensure that you use the latest supported privacyIDEA version 3.9.2 and then upgrade to eduMFA
+> 
+> These migration steps are applicable for the versions 1.0.0 to 1.5.1
+
+* Uninstall privacyIDEA and stop your nginx or `Apache2` or `nginx` service
+* Move/Copy all configurations from `/etc/privacyidea/` to `/etc/edumfa/`
+    * The config file needs to be renamed from `pi.cfg` to `edumfa.cfg`
+    * Rename/Replace `PI_` in the configuration file with `EDUMFA_` (e.g. `PI_ENCFILE` changes to `EDUMFA_ENCFILE`)
+    * Update the paths in the configuration file to `/etc/edumfa/`
+    * Update the log file path to `/var/log/edumfa/edumfa.log`
+* Install eduMFA (e.g. using Container, PIP or the `.deb` Package)
+    * When using the `.deb` package, make sure to not overwrite the existing configuration file
```

Comment:
how? will the config be overwritten in any case?

Reply:
When having a config present, during installation you'll be asked if you like to keep the current config or use the maintainers version.

Reply:
```suggestion
    * When using the `.deb` package, make sure to not overwrite the existing configuration file (answer N or O when asked by dpkg to keep your installed version)
```

Reply:
I think the current explanation should be mostly sufficient. Maybe we can add `during installation` and/or `when prompted`

Reply:
OK
```suggestion
    * When using the `.deb` package, make sure not to overwrite the existing configuration file when prompted
```